index.html

```diff
@@ -46,17 +46,17 @@
   </head>
   <body>
     <div class="card">
-      <img src="my-profile-picture.png" alt="Girlcoder" style="width: 100%" />
-      <h1>Zandile Doe</h1>
+      <img src="me.png" alt="My name" style="width: 100%" />
+      <h1>Omolemo Lethuloe</h1>
       <p class="title">Software developer</p>
-      <p>Girlcode Bootcamp</p>
+      <p>Girlcode Software Development Learnership</p>
       <div style="margin: 24px 0">
         <a href="https://twitter.com/GirlCode_za"><i class="fa fa-twitter"></i></a>
         <a href="https://www.linkedin.com/company/girlcodeza"><i class="fa fa-linkedin"></i></a>
         <a href="https://github.com/GirlCodeZA"><i class="fa fa-github"></i></a>
       </div>
       <p>
-        <a href="mailto:girlcode@gmail.com"><button>Contact</button></a>
+        <a href="olethuloe@gmail.com"><button>Contact</button></a>
       </p>
     </div>
   </body>
```

Comment:
Same here, your personal `index.html` should be in your personal repo, not this one :)